app/qml/components/MMCodeScanner.qml

```diff
@@ -54,27 +54,16 @@ Drawer {
     }
   }
 
-  Canvas {
-    id: canvas
-
-    anchors.fill: parent
-    opacity: 0.8
-
-    onPaint: {
-      var ctx = getContext("2d");
-      let w = parent.width
-      let h = parent.height
-      ctx.fillStyle = __style.nightColor
-      ctx.fillRect(0, 0, parent.width, parent.height);
-      ctx.fill();
-      if(parent.width < parent.height)
-        ctx.clearRect(w / 4, h / 2 - w / 4, w / 2, w / 2)
-      else
-        ctx.clearRect(w / 2 - h / 4, h / 4, h / 2, h / 2)
-    }
```

Comment:
Why this is removed? I think we need to blur/darken the background image, so keep it there (?)